TestGetFileContent test is failing when running go test, on Mac

```
time="2021-05-26T16:26:29+05:30" level=info msg=Trace args="[git clean -fdx]" dir="/var/folders/55/y1ln_mzn4cg9f605bphmkgbc0000gr/T/https:__github.com_argoproj_argocd-example-apps_" operation_name="exec git" time_ms=14.280113
--- FAIL: TestGetFileContent (5.86s)
    --- FAIL: TestGetFileContent/pull_an_invalid_file (1.23s)
        repo_service_test.go:348: 
                Error Trace:    repo_service_test.go:348
                Error:          Error message not equal:
                                expected: "open /tmp/https:__github.com_argoproj_argocd-example-apps_/this-file-does-not-exist.md: no such file or directory"
                                actual  : "open /var/folders/55/y1ln_mzn4cg9f605bphmkgbc0000gr/T/https:__github.com_argoproj_argocd-example-apps_/this-file-does-not-exist.md: no such file or directory"
                Test:           TestGetFileContent/pull_an_invalid_file
FAIL
coverage: 79.4% of statements
FAIL    github.com/argoproj-labs/applicationset/pkg/services    17.205s
ok      github.com/argoproj-labs/applicationset/pkg/services/scm_provider       7.943s  coverage: 65.2% of statements
ok      github.com/argoproj-labs/applicationset/pkg/utils       3.015s  coverage: 71.2% of statements
FAIL
make: *** [test] Error 1
```

--- a/pkg/services/repo_service_test.go
+++ b/pkg/services/repo_service_test.go
@@ -3,6 +3,7 @@ package services
 import (
 	"context"
 	"errors"
+	"os"
 	"sort"
 	"strings"
 	"testing"
@@ -319,7 +320,7 @@ func TestGetFileContent(t *testing.T) {
 			repoURL:       "https://github.com/argoproj/argocd-example-apps/",
 			revision:      commitID,
 			path:          "/this-file-does-not-exist.md",
-			expectedError: errors.New("open /tmp/https:__github.com_argoproj_argocd-example-apps_/this-file-does-not-exist.md: no such file or directory"),
+			expectedError: errors.New("open " + os.TempDir() + "/https:__github.com_argoproj_argocd-example-apps_/this-file-does-not-exist.md: no such file or directory"),
 		},
 	}
 

